src/databricks/sql/client.py

```diff
@@ -251,6 +254,7 @@ def __init__(
         thrift_backend: ThriftBackend,
         result_buffer_size_bytes: int = DEFAULT_RESULT_BUFFER_SIZE_BYTES,
         arraysize: int = DEFAULT_ARRAY_SIZE,
+        lz4_compression: bool = True,
```

Comment:
@moderakh

Reply:
@mohitsingla-db similar to the JDBC/ODBC driver this should be passed to the client upfront, rather than per cursor or per query

```
sql.connect(host, httpath, your_new_option)
```

`your_new_option` should be passed as kwargs